### ISSUE TITLE
Camera's BUG: when I set elev=90 or 270

### ISSUE THREAD

**Issue by qmlu-cn** [facebookresearch/pytorch3d#406]
when I set elev = 90 or 270, the result is a line.
```
self.elev=90
self.azim=0
R, T = look_at_view_transform(self.dist, self.elev, self.azim, device=self.device,at=at) 
self.cameras = FoVPerspectiveCameras(device=self.device, R=R, T=T)
std_image = self.renderer(std_mesh, cameras=self.cameras)
```
![90](https://user-images.githubusercontent.com/26488213/96582892-81052500-130e-11eb-8882-109814f012da.png)

**Comment by albertotono** [comment]
Do you have this warning?

```
 /usr/local/lib/python3.6/dist-packages/pytorch3d/transforms/transform3d.py:707: UserWarning: R is not a valid rotation matrix
  warnings.warn(msg)
```

**Comment by nikhilaravi** [comment]
Which version of PyTorch3D are you using? Are you installing from conda or Github?

**Comment by qmlu-cn** [comment]
> Do you have this warning?
> 
> ```
>  /usr/local/lib/python3.6/dist-packages/pytorch3d/transforms/transform3d.py:707: UserWarning: R is not a valid rotation matrix
>   warnings.warn(msg)
> ```
> 
> as workaround I used 89.9999

Yes

**Comment by qmlu-cn** [comment]
> Which version of PyTorch3D are you using? Are you installing from conda or Github? This issue should have been fixed by [eb517dd](https://github.com/facebookresearch/pytorch3d/commit/eb517dd70b20c8431a1cfc46e47b4c5c78529c5b). One possibility is that you need to increase the tolerance on this line:
> 
> https://github.com/facebookresearch/pytorch3d/blob/30e4e891dbbccb6b9ca416182e70187eaced71a0/pytorch3d/renderer/cameras.py#L1232

I'm using PyTorch3D 2.5. And I installed it from conda. Thank you for your reply.  I have fixed this problem.
